src/store/options.js

```diff
@@ -34,10 +35,17 @@ const Options = {
       await chrome.storage.local.set({ options });
 
       // Send update message to another contexts (background page / panel / options)
-      await chrome.runtime.sendMessage({
-        action: UPDATE_OPTIONS_ACTION_NAME,
-        options,
-      });
+      try {
+        chrome.runtime.sendMessage({
```

Comment:
Is it intentional to drop the await here and make it fire-and-forget?

Reply:
Yes, we have to do this, as I discovered, that if the background process runs without another cotexts, this chrome.runtime.sendMessage()` throws an unexpected error (on one of the browsers, Safari or Chrome, I don't remember which). It is weird behavior, but the code should not break in that place, as then options are not set correctly. The call is only to notify other open contexts (in our case opened panel or options page, or the background process) that options have changed.